lib/ontologies_linked_data/concerns/semantic_artefact/attribute_fetcher.rb

```
@@ -0,0 +1,67 @@
+                        mapping = self.class.attribute_mappings[attr]
+                        next unless mapping
+                        model = mapping[:model]
+                        mapped_attr = mapping[:attribute]
+                        hash[model][attr] = mapped_attr
+                    end
+
+                    populate_from_self(grouped_attributes[self.class]) if grouped_attributes[self.class].any?
+                    fetch_from_ontology(grouped_attributes[:ontology]) if grouped_attributes[:ontology].any?
+                    fetch_from_submission(grouped_attributes[:ontology_submission]) if grouped_attributes[:ontology_submission].any?
+                    fetch_from_metrics(grouped_attributes[:metric]) if grouped_attributes[:metric].any?
+                end
```

RuboCop (1.73)

[convention] Assignment Branch Condition size for bring is too high. [<10, 29, 6> 31.26/23]

(Metrics/AbcSize)